docs/_includes/head.htm

```diff
@@ -2,15 +2,17 @@
 <meta charset="utf-8" />
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 <meta name="viewport" content="width=device-width, initial-scale=1" />
-<link href="{{ "/css/style.css" | prepend:site.baseurl }}" rel="stylesheet" />
+<link href="/css/style.css" rel="stylesheet" />
```

Comment:
We need this to fix the menus so that you can navigate back and forth across versions!

Reply:
Also, we MUST reuse the common JS, otherwise this whole thing doesn't work. The JS file is the magic that makes this work at all.

Reply:
Also note, you'll never have to make all the changes I made again. I built a framework where all you need to do is copy the generated site contents (minus that one var we're gonna add) and you're done.

Reply:
My point is that each releases/<version> folder holds its own css folder and so the pages in the releases/<verison> folder should use the releases/<version>/css folder for its css. Otherwise we restrict our future selves in how we an modify the css in master to reskin the site.